scripts.py

```diff
@@ -195,27 +195,18 @@ def __init__(self, providers):
             self.providers.append(i)
 
     def do_run(self):
-        offsets = dict()
```

Comment:
I wasn't sure about this but looking at the implementation of `provider.run_once_and_update_timestamp` I think it makes sense.